### PR TITLE
Capture stream data

### PR DESCRIPTION
**Pull request by podviaznikov**

### CODE REVIEW

File: lib/monitor.js [hunk 1]
```diff
@@ -132,6 +132,25 @@ Monitor.prototype.stopSocketsMonitor = function () {
 };
 
 /**
+ * Capture stream data.
+ */
+Monitor.prototype.captureStream = function (streamName, stream) {
```
Comment (bkendall) [inline lib/monitor.js]: `captureStream` seems misleading, since its' not actually catching data but just stream events... maybe `captureStreamEvents`?

Comment (podviaznikov) [inline lib/monitor.js]: renamed

**Comment by bkendall** [comment]
lerr gerr t'meee

**Comment by bkendall** [comment]
:+1:

**Pull request merged**